contentcuration/contentcuration/frontend/channelEdit/pages/StagingTreePage/index.vue

```diff
@@ -385,11 +392,21 @@
       },
     },
     watch: {
-      nodeId(newNodeId) {
-        this.elevated = false;
-        this.loadAncestors({ id: newNodeId });
-        this.loadChildren({ parent: newNodeId, root_id: this.stagingId });
-      },
+  nodeId(newNodeId) {
+    this.elevated = false;
+    this.isLoadingchild = true;
+
+    Promise.all([
+      this.loadAncestors({ id: newNodeId }),
+      this.loadChildren({ parent: newNodeId, root_id: this.stagingId }),
+      this.loadCurrentChannel({ staging: true }),
+    ]).then(() => {
+      this.isLoadingchild = false;
+      this.loadCurrentChannelStagingDiff();
+    }).catch(error => {
+      throw new Error(error);
+    });
```

Comment:
The default behavior of a promise chain without a catch would be to throw the error, so you can omit this `catch`